### ISSUE TITLE
代码问题

### ISSUE THREAD

**Issue by JEDA836** [yuliu316316/VDMUFusion#2]
你好，在train_losses函数中，img_f只是由mask和x1_noisy,x2_noisy得到，而在文章Fig.2中，得到F(t-1)还需要F(t)，这里是不是代码和图不统一
    def train_losses(self, model, sourceImg1, sourceImg2, t, loss_scale):
        sourceImg2 = F.interpolate(sourceImg2, scale_factor=0.25, mode='bicubic', align_corners=False)
        sourceImg2 = F.interpolate(sourceImg2, scale_factor=4, mode='bicubic', align_corners=False)
        noise = torch.randn_like(sourceImg1)
        x1_noisy, x2_noisy = self.q_sample(sourceImg1, sourceImg2, t, noise=noise)
        input = torch.cat([sourceImg1, sourceImg2, x1_noisy, x2_noisy], dim=1)
        predicted_noise, mask = model(input, t)
        imgf = mask * x1_noisy + ((torch.ones(mask.shape).to(device) - mask) * x2_noisy)
        noise_loss = loss_scale * torch.norm(noise-predicted_noise, p=1) /(256*256)
        fusion_loss = (torch.norm(imgf-x1_noisy, p=1) + torch.norm(imgf-x2_noisy, p=1)) / (256*256)
        assert predicted_noise.shape == noise.shape
        return noise_loss, fusion_loss
